src/EditorFeatures/Core/InlineRename/InlineRenameSession.cs

```diff
@@ -389,6 +389,11 @@ private void OnWorkspaceChanged(object sender, WorkspaceChangeEventArgs args)
         {
             if (args.Kind != WorkspaceChangeKind.DocumentChanged)
             {
+                Logger.Log(FunctionId.Rename_InlineSession_Cancel_WorkspaceChanged, KeyValueLogMessage.Create(m =>
```

Comment:
While I was here I've seen bugs about rename dismissing without any user interaction. I _believe_ this is the cause, but this will help track down more accurately.